### DISCUSSION TITLE
Meeting list by day

### DISCUSSION THREAD

**Discussion by trivalleyaa** [code4recovery/12-step-meeting-list#730] [Q&A]
My members have asked if there is a way to have the meeting list populate for the day (weekday) it is opened. Right now it seems to populate by the last day it was opened. And when I clicked on the dropdown to select my today, it did not go anywhere when I clicked on Enter. Any help would be appreciated.

**Comment by tech2serve** [comment]
Hi @trivalleyaa,

Is this your website? https://trivalleyaa.org/

The expected behavior is for the current day's meetings to be shown. Check out our [demo site](https://demo.code4recovery.org/meetings) for TSML.

When a website shows the incorrect day, as yours is doing, it is usually stems from one of two causes:

1) Server side cache needs to be cleared,
2) There is an error on the site or some other plugin causing the problem. 

For the first one, please work with your hosting provider to determine how to clear the cache if you don't already know how. If the day then shows properly, you know you've found the source of the problem, but you may need to work on finding the real cause. For one site I know of, the problem was the plugin the hosting company used for caching. 

To explore the second possibility, please open a dev console and watch for errors, and then try and resolve them. I do see missing graphics on your home page, but not too much exciting on the meeting page so...

Please start with these two steps, and keep us posted.
